src/components/AppBar.tsx

```diff
@@ -0,0 +1,96 @@
+import { Hambuger, RpcSwitcher } from "../components";
+import { WalletMultiButton } from "@solana/wallet-adapter-react-ui";
+import { ReactSVG } from "react-svg";
+import { useAtom } from "jotai";
+import { themeAtom, selectedPoolAtom } from "stores/globalStates";
+import Image from "next/image";
+import { usePoolsList } from "hooks/usePoolsList";
+import { formatAddress } from "utils/formatUtils";
+
+const AppBar = ({}) => {
+  const [selectedPool, setSelectedPool] = useAtom(selectedPoolAtom);
+  const [theme, setTheme] = useAtom(themeAtom);
+  const { poolList, isLoading, isError } = usePoolsList();
+
+  if (isLoading) return <div></div>;
```

Comment:
Not sure if this is available in the design system used but nice to have a generic skeleton component for stuff like this https://ant.design/components/skeleton/